src/server.rs

```diff
@@ -271,9 +394,16 @@ pub enum ServerFlowEvent {
         /// Formerly enqueued [`Response`] that was now sent.
         response: Response<'static>,
     },
-    CommandReceived {
-        command: Command<'static>,
+    /// Command received.
+    CommandReceived { command: Command<'static> },
+    /// Command AUTHENTICATE received.
```

Comment:
This doc should mention that either `authenticate_continue` or `authenticate_finish` should be called afterwards.